src/models/CityModel.js

```diff
@@ -8,10 +8,14 @@ class CityModel {
   _extrasEnabled: boolean
   _sortingName: string
   _prefix: ?string
+  _latitude: number | null
+  _longitude: number | null
+  _aliases: { [alias: string]: {| long: number, lat: number |} } | null
 
   constructor (params: {|
     name: string, code: string, live: boolean, eventsEnabled: boolean, extrasEnabled: boolean,
-    sortingName: string, prefix: ?string
+    sortingName: string, prefix: ?string, latitude: number | null, longitude: number | null,
+    aliases: { [alias: string]: {| long: number, lat: number |} } | null
```

Comment:
Minor: Probably better to name the params 'longitude' instead of 'long' to stay consistent.